libr/include/r_cons.h

```diff
@@ -421,6 +421,17 @@ typedef struct r_cons_canvas_t {
 #define UTF_CIRCLE "\u25EF"
 #define UTF_BLOCK "\u2588"
 
+// Emoji
+#define UTF8_POLICE_CARS_REVOLVING_LIGHT "🚨"
```

Comment:
hahahah i was expecting names like EMOJI_FIX, EMOJI_IGNORE, ...

also, are we sure that all compilers handle emoji chars properly in C strings? <- i can answer myself, that even tcc handles this fine

Reply:
> hahahah i was expecting names like EMOJI_FIX, EMOJI_IGNORE, ...

not sure whether `#define EMOJI_BROKEN UTF8_SKULL_AND_CROSSBONES` is actually appropriate 👀

> also, are we sure that all compilers handle emoji chars properly in C strings? <- i can answer myself, that even tcc handles this fine

yeah if you open the include file in a hex editor (such as r2), the emoji will just consist of their UTF8 bytes ... so unless the compiler can't handle chars > 0x7f in a string, it should be just fine

Reply:
> even tcc handles this fine

Doesn't mean msvc handles it fine.

Reply:
I am testing with msvc right now and I can confirm that it handles it fine.

Reply:
Besides, i don't think there is any 7-bit [Windows code page](https://en.wikipedia.org/wiki/Windows_code_page).